master/reference/node/configuration.md

```diff
@@ -32,7 +32,7 @@ The `calico/node` container is primarily configured through environment variable
 | CALICO_IPV6POOL_NAT_OUTGOING | Controls NAT Outgoing for the IPv6 Pool created at start up. [Default: `false`] | boolean |
 | CALICO_STARTUP_LOGLEVEL      | The log severity above which startup calico/node logs are sent to the stdout. [Default: `ERROR`] | DEBUG, INFO, WARNING, ERROR, CRITICAL, or NONE (case-insensitive) |
 | CLUSTER_TYPE | Contains comma delimited list of indicators about this cluster.  e.g. k8s, mesos, kubeadm, canal, bgp | string |
-| ETCD_ENDPOINTS    | A comma separated list of etcd endpoints [Default: `http://127.0.0.1:2379`] (optional) | string |
+| ETCD_ENDPOINTS    | A comma separated list of etcd endpoints [Default: `http://127.0.0.1:2379`] (required) | string |
```

Comment:
Since this is now a required field, I'm not sure if we need the `[Default:...]` portion anymore since the value is entirely dependent on what the user inputs. WDYT?

Reply:
Yes, it's a good point!